app/controllers/shows_controller.rb

```diff
@@ -0,0 +1,7 @@
+include ShowsHelper
```

Comment:
not here, this is global, put it bellow `ShowsController`, it should be in that class